jest.config.ts

```diff
@@ -0,0 +1,10 @@
+// import type { Config } from '@jest/types';
+
+module.exports = {
+  preset: 'ts-jest',
+  transform: {
+    '^.+\\.(ts|tsx)?$': 'ts-jest',
+    '^.+\\.(js|jsx)$': 'babel-jest'
```

Comment:
Thanks for setting this up! It'll probably be a really helpful guide when we set up frontend testing too. Since this is just the API repo, do we actually need to worrk about tsx/jsx files?

Reply:
No I don't think we do tbh. I mostly copied and pasted from a sample one

Reply:
So do we actually need this? Doesn't seem to make sense to leave it in unless we plan on adding react files to this repo

Reply:
Nah, I'll remove this as well once I merge it and ensure that doesn't break things

Reply:
opened new PR for this work